CONTRIBUTING.md

```diff
@@ -46,32 +51,29 @@ be able to work on TabPy changes:
     cd TabPy
     ```
 
-Before making any code changes run environment setup script.
-For Windows run this command from the repository root folder:
+4. Registeg TabPy repo as a pip package:
 
-```sh
-utils\set_env.cmd
-```
+    ```sh
+    pip install -e .
```

Comment:
missing argument (pip install tabpy -e  ?)

Reply:
no, this is actually correct - pip discovers package in the current folder as it has setup.py now.

Reply:
TIL: Sweet!